packages/editor-ui/src/utils/canvasUtils.ts

```diff
@@ -13,47 +13,40 @@ import type { Route } from 'vue-router';
 	'@/utils'.
 */
 
-export const scaleSmaller = ({ scale, offset: [xOffset, yOffset] }: IZoomConfig): IZoomConfig => {
-	scale /= 1.25;
-	xOffset /= 1.25;
-	yOffset /= 1.25;
-	xOffset += window.innerWidth / 10;
-	yOffset += window.innerHeight / 10;
-
-	return {
-		scale,
-		offset: [xOffset, yOffset],
-	};
+const SCALE_INCREASE_FACTOR = 1.25;
+const SCALE_DECREASE_FACTOR = 0.75;
+const MIN_SCALE = 0.2;
+const MAX_SCALE = 10;
```

Comment:
Could we perhaps even do with a MAX_SCALE of 5? It would look like this when fully zoomed in: 
<img width="1383" alt="Screenshot 2023-10-07 at 8 22 23" src="https://github.com/n8n-io/n8n/assets/299644/de0941b8-ff3e-4503-9971-147819f74e1d">

I couldn't think of it being useful past this.